sdk/iot/Azure.Iot.Hub.Service/tests/CustomRequestSanitizer.cs

```diff
@@ -12,13 +12,15 @@ namespace Azure.Iot.Hub.Service.Tests
     internal class CustomRequestSanitizer : RecordedTestSanitizer
     {
         internal const string FakeHost = "FakeHost.net";
+        internal const string FakeStorageUri = "https://fake.blob.core.windows.net";
 
         public CustomRequestSanitizer()
            : base()
         {
             // Sanitize SAS tokens in request body
             JsonPathSanitizers.Add("outputBlobContainerUri");
             JsonPathSanitizers.Add("inputBlobContainerUri");
+            JsonPathSanitizers.Add("symmetricKey");
```

Comment:
There are few more things we are storing in our recorded files. I will be sanitizing all of them in the next PR.